bin/grade_item.py

```diff
@@ -259,11 +259,22 @@ def just_grade_item(next_directory,next_to_grade,which_untrusted):
         subprocess.call(['ls', '-lR', checkout_path], stdout=open(tmp_logs + "/overall.txt", 'a'))
 
 
+    # --------------------------------------------------------------------
+    # START DOCKER
+
+    container = subprocess.check_output(['docker', 'run', '-t', '-d',
+                                         '-v', tmp + ':' + tmp,
+                                         '-v', '/usr/local/submitty:/usr/local/submitty',
+                                         '-v', '/var/local/submitty:/var/local/submitty',
```

Comment:
consider changing `/usr/local/submitty:/usr/local/submitty` and `/var/local/submitty:/var/local/submitty` to `/usr/local/submitty:/usr/local/submitty:ro` and `/var/local/submitty:/var/local/submitty:ro` (https://stackoverflow.com/a/20317879/3646475)